app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -3510,6 +3523,89 @@ class BrowserTabViewModelTest {
         assertNull(privacyGradeState().privacyGrade)
     }
 
+    fun whenEditBookmarkRequestedThenDaoIsNotUpdated() {
+        coroutineRule.runBlocking {
```

Comment:
can you update this to`whenEditBookmarkRequestedThenDaoIsNotUpdated() = coroutineRule.runBlocking` to follow the convention of the other tests?